designer/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala

```diff
@@ -155,6 +148,7 @@ class DefaultComponentService private(componentLinksConfig: ComponentLinksConfig
       .getOrElse(componentLinks)
   }
 
+  // TODO: We should log some errors when we find some differences between definition of components
```

Comment:
I think it's not needed because validation is already performed in `ComponentsValidator`.

Reply:
I forgot about it - I replaced this comment with comment about where it can be found